package-scripts.js

```diff
@@ -6,13 +6,10 @@ module.exports = {
     },
     test: {
       default: {
-        description: 'Runs AVA with nyc (which is configured in package.json)',
-        script: 'cross-env NODE_ENV=test nyc ava',
-      },
-      watch: {
-        description: 'Run AVA in watch mode',
-        script: 'ava -w --require babel-register',
+        description: `This runs jest with coverage. If we're on Travis, then we'll ignore the cache (just in case).`,
+        script: `jest --coverage ${process.env.CI ? '--no-cache' : ''}`,
       },
+      watch: 'jest --watch --coverage',
```

Comment:
<3 thank you. We'll only make it better. I want Jest to be visual and run inside Electron. That's a fun hackathon project I was thinking about.